### PR TITLE
IllusionFpv freestyle rates

### PR DESCRIPTION
**Pull request by IllusionFpv**
Screenshot 2021-12-09 19 53 11

### CODE REVIEW

**Comment by sugaarK** [comment]
want to add in pictures same as the other rates profile and maybe reproduce the rates in the same way with actual as an option?

**Comment by IllusionFpv** [comment]
Sure, I can add pictures and betaflight ones as option if is possible

Il gio 9 dic 2021, 01:29 sugaarK ***@***.***> ha scritto:

File: presets/4.3/rates/illusionfpv_freestyle.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#$ TITLE: freestyle rates by IllusionFpv
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: freestyle rates IllusionFpv quick
+#$ AUTHOR: IllusionFpv
+#$ DESCRIPTION: Freestyle rates by IllusionFpv
+#$ DESCRIPTION: Quick rates from developer IllusionFpv, with 1000 deg/sec on roll, 950 deg/sec on pitch and 700 deg/sec on yaw 
+
```
Comment (sugaarK) [inline presets/4.3/rates/illusionfpv_freestyle.txt:9]: missing `#$ DISCUSSION: https://github.com/betaflight/firmware-presets/
pull/106`

**Review by sugaarK — CHANGES_REQUESTED** [review]

**Comment by sugaarK** [comment]
> 

remake in actual if you can that the new default and much easier to work with

File: presets/4.3/rates/illusionfpv_freestyle.txt [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#$ TITLE: freestyle rates by IllusionFpv
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: freestyle rates IllusionFpv quick
```
Comment (limonspb) [inline presets/4.3/rates/illusionfpv_freestyle.txt:5]: key words should be comma-separated, please

**Review by limonspb — CHANGES_REQUESTED** [review]

File: presets/4.3/rates/illusionfpv_freestyle.txt (hunk 2 above)
Comment (IllusionFpv) [reply presets/4.3/rates/illusionfpv_freestyle.txt]: added some commas

File: presets/4.3/rates/illusionfpv_freestyle.txt (hunk 1 above)
Comment (IllusionFpv) [reply presets/4.3/rates/illusionfpv_freestyle.txt:10]: fixed

**Comment by IllusionFpv** [comment]
> > 
> 
> remake in actual if you can that the new default and much easier to work with

Quick rates are easy to work too, no need to ad options

**Review by sugaarK — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Pull request merged**